apps/fifty-fifty-php/app/Providers/AppServiceProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class AppServiceProvider extends ServiceProvider
+{
+    /**
+     * Register any application services.
+     */
+    public function register(): void
+    {
+        //
```

Comment:
ok, I'm convinced now that this is meant to be, no more comments about them 😂

Reply:
All those empty lines were added by scaffolded laravel app, no special purpose there 😄